model/user.go

```diff
@@ -80,6 +80,7 @@ type User struct {
 	MfaActive          bool      `json:"mfa_active,omitempty"`
 	MfaSecret          string    `json:"mfa_secret,omitempty"`
 	LastActivityAt     int64     `db:"-" json:"last_activity_at,omitempty"`
+	IsBot              bool      `db:"-" json:"is_bot"`
```

Comment:
Just to confirm, `IsBot` is only set for `model.User` made from Bot objects. These Users are never persisted to the db, so  `db:"-"`?

Reply:
The `Users` are persisted, but the `IsBot` column is never persisted.